pytorch_lightning/plugins/precision/ipu.py

```diff
@@ -30,6 +30,8 @@ class IPUPrecisionPlugin(PrecisionPlugin):
     """Precision plugin for IPU integration."""
```

Comment:
docstring should include that it raises a misconfiguration error if the precision value is not supported

Reply:
Updated!